protocol/triple/health/triple_health/health.triple.go

```diff
@@ -66,24 +68,27 @@ var (
 	_ Health_WatchServer = (*HealthWatchServer)(nil)
 )
 
-// Health is a client for the grpc.health.v1.Health service.
+// Health is a client for the dubbo.health.v1.Health service.
 type Health interface {
 	Check(ctx context.Context, req *HealthCheckRequest, opts ...client.CallOption) (*HealthCheckResponse, error)
 	Watch(ctx context.Context, req *HealthCheckRequest, opts ...client.CallOption) (Health_WatchClient, error)
 }
 
-// NewHealth constructs a client for the grpc.health.v1.Health service.
+// NewHealth constructs a client for the dubbo.health.v1.Health service.
 func NewHealth(cli *client.Client, opts ...client.ReferenceOption) (Health, error) {
 	conn, err := cli.Dial("grpc.health.v1.Health", opts...)
```

Comment:
This should be ```DialWithInfo```. Please help me to modify this part.

Reply:
ok